timers.js

```diff
@@ -1,4 +1,27 @@
 
+$(document).ready(function () {
 
+  seconds = 0;
+  timerID = "";
 
+  $("#start").on("click",doStart);
```

Comment:
Right now if I click on the timer 10 times, it will start 10 different timers (i.e., the timer on-screen will increment more than one number per second). How would you go about preventing this from happening?